integration_tests/suite/helpers/database.py

```diff
@@ -107,8 +136,6 @@ def insert_call_log_participant(self, **kwargs):
     def find_last_call_log(self):
         session = self.Session()
         call_log = session.query(CallLog).order_by(CallLog.date).first()
-        if call_log:
-            make_transient(call_log)
```

Comment:
Can you extract your modification into a separate commit to explain why it was added and why you need to remove?

Reply:
Sure, once you make an alchemy object transient, lazy/sub_query relationships don't work anymore.

But it should be the reverse any usage of make_transient() should be commented as it's not recommended to use that method and often hide an other issue.

Reply:
Agreed that usage of make_transient should be commented or not used.
In this case, I'm just surprised that you can remove it without side effect (or you fix something that I did'nt see?)

I'm not sure to understand the difference between `with self.Session()` instead `session.commit()`. Have you some documentation about the Session context manager?

Reply:
> Agreed that usage of make_transient should be commented or not used.
> In this case, I'm just surprised that you can remove it without side effect (or you fix something that I didn't see?)

I have ran test in separate commit without issue, so...

> I'm not sure to understand the difference between `with self.Session()` instead `session.commit()`. Have you some documentation about the Session context manager?

I was thinking of these:
* https://github.com/wazo-pbx/wazo-webhookd/blob/master/wazo_webhookd/plugins/subscription/service.py#L41
* https://github.com/wazo-pbx/wazo-call-logd/blob/master/wazo_call_logd/database/dao.py#L47